test/node-http-test.js

```diff
@@ -94,10 +96,133 @@ describe('Node HTTP', function() {
     await node.close();
   });
 
+  describe('JSON API', function () {
+    it('should get balance by address', async () => {
+      // Generate new cbAddress.
+      const cType = network.keyPrefix.coinType;
+      const nType = network.type;
+      const master = HDPrivateKey.generate();
+      const key = master.derivePath(`m/44'/${cType}'/0'/0/0`);
+      const cbAddress = Address.fromPubkey(key.publicKey).toString(nType);
+      const keyring = KeyRing.fromPrivate(key.privateKey);
```

Comment:
general note: I think you could deploy a `MemWallet` here to simplify this stuff. See `chain-test.js`